aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -712,7 +712,7 @@ TLS restrictions for HTTP/2:
 * Minimum ephemeral key exchange sizes:
   * Elliptic curve Diffie-Hellman (ECDHE) &lbrack;[RFC4492](https://www.ietf.org/rfc/rfc4492.txt)&rbrack;: 224 bits minimum
   * Finite field Diffie-Hellman (DHE) &lbrack;`TLS12`&rbrack;: 2048 bits minimum
-* Cipher suite not blacklisted
+* Cipher suite not exclude listed. 
```

Comment:
```suggestion
* Cipher suite not prohibited. 
```

Reply:
Consistency ... none of the entries in the list are complete sentences with a period.